.github/workflows/release-manual.yml

```diff
@@ -0,0 +1,169 @@
+        description: "set the new release as a draft instead of published"
+        type: boolean
+        required: false
+        default: false
+      prerelease:
+        description: "set to true for nightly builds"
+        type: boolean
+        required: false
+        default: true
+
+jobs:
+  release_vars:
+    runs-on: ubuntu-latest
+    outputs:
+      RELEASE_SHA: ${{ steps.set-vars.outputs.RELEASE_SHA }}
+      SHORT_REVISION: ${{ steps.set-vars.outputs.SHORT_REVISION }}
+      RELEASE_VERSION: ${{ steps.set-vars.outputs.RELEASE_VERSION }}
+      IMAGE: ${{ steps.set-vars.outputs.IMAGE }}
+      GHC_VERSION: ${{ steps.set-ghc-version.outputs.GHC_VERSION }}
+    steps:
+    # if this plugin is bad we just override it
+#       - name: Bump version and push tag
+#         id: tag_version
+#         uses: mathieudutour/github-tag-action@v6.1
+#         with:
+#           github_token: ${{ secrets.GITHUB_TOKEN }}
+#           default_bump: ${{ inputs.tag_default_bump }}
+#           tag_prefix: ""
+#           commit_sha: ${{ inputs.release_sha }}
+
+          
+      - name: Set variables for release
+        id: set-vars
+        run: |
+          RELEASE_SHA=${{ github.sha }}
+          if [[ ! -z ${{ inputs.release_sha }} ]]; then
+            RELEASE_SHA=${{ inputs.release_sha }}
+          fi
+          echo "RELEASE_SHA=$RELEASE_SHA" >> $GITHUB_OUTPUT
+          SHORT_REVISION=$(echo $RELEASE_SHA | cut -c 1-7)
+          echo "SHORT_REVISION=$SHORT_REVISION" >> $GITHUB_OUTPUT
+          echo "RELEASE_VERSION=${{ inputs.release_version }}" >> $GITHUB_OUTPUT
+          echo "IMAGE=ghcr.io/kadena-io/pact-5:sha-$SHORT_REVISION" >> $GITHUB_OUTPUT
+          
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ steps.set-vars.outputs.RELEASE_SHA }}
+          sparse-checkout: |
+            .github/workflows/applications.yml
+        
+      - name: Get ghc version
+        id: set-ghc-version
+        run: |
+          VERSION=$(grep -Po '(?<="ghc": \[")(\d\.\d\.\d)' .github/workflows/applications.yml | head -1)
+            if [[ -z $VERSION ]]; then
+              echo "Unable to get version from pact build"
+              exit 1
+            fi
+          echo "GHC_VERSION=$VERSION" >> $GITHUB_OUTPUT
+
+  release_to_github:
+    runs-on: ubuntu-latest
+    needs: release_vars
+    env:
+      SHORT_REVISION: ${{ needs.release_vars.outputs.SHORT_REVISION }}
+      RELEASE_VERSION: ${{ needs.release_vars.outputs.RELEASE_VERSION }}
+      GHC_VERSION: ${{ needs.release_vars.outputs.GHC_VERSION }}
+      IMAGE: ${{ needs.release_vars.outputs.IMAGE }}
+    steps:
+    
+      - name: Get release artifacts
+        run: |
+          UBUNTU_TAR=pact.${{ env.GHC_VERSION }}.ubuntu-22.04.${{ env.SHORT_REVISION }}.tar.gz
+          MACOS_TAR=pact.${{ env.GHC_VERSION }}.macos-14.${{ env.SHORT_REVISION }}.tar.gz
+          
+          UBUNTU_RELEASE_FILE=pact-binary-bundle.ubuntu-22.04.${{ env.RELEASE_VERSION }}.tar.gz
```

Comment:
would help a lot if the binary name format is consistent across platform/arch/os eg nodejs follows this `node-v<version>-<os>-<arch>.<tarball_format>` eg `node-v20.17.0-linux-x64.tar.xz` , keep in mind there are 3 or more projects that relay on github releases being consistent and reliable like  [pactup](https://github.com/kadena-community/pactup), [pact-vscode](https://github.com/kadena-community/pact-vscode) , [pact-zed](https://github.com/kadena-community/pact-zed) and [pact-toolbox](https://github.com/kadena-community/pact-toolbox)

Reply:
for more examples you can also check https://nodejs.org/en/download/prebuilt-binaries if you want to follow this format

Reply:
I'll have it match the other releases

Reply:
it matches the way we have named pact releases
pact-VERSION-linux-22.04
pact-VERSION-aarch64-osx